taxonomies/categories.txt

```diff
@@ -19346,16 +19491,26 @@ fr:Jelly beans
 nl:Jelly beans
 zh:软心豆粒糖
 
+<en:Gelified candies
+en:Jelly candy
+fr:Bonbon gélifié
+ciqual_food_code:en:31060
+ciqual_food_name:en:Jelly candy
+ciqual_food_name:fr:Bonbon gélifié
+
 <en:Candies
-en:Lollipops
+en:Lollipops, Hard candy, Lollipop
```

Comment:
Same here, we don't want to make hard candy a synonym of lollipop.

Reply:
Roger that, je duplique. Merci !